installer_scripts/install-yb-voyager

```diff
@@ -28,7 +28,7 @@ GO_VERSION="go1.18"
 GO="$GOROOT/bin/go"
 
 LOG_FILE=/tmp/install-yb-voyager.log
-ORA2PG_VERSION="23.1"
+ORA2PG_VERSION="24a4761fdd2d0e6f167807e3fba0576c5f0af491"
```

Comment:
Add a comment stating that the commit-hash is from `main` branch after the release 23.1.

Reply:
👍